packages/core-contracts/test/proxy/BeaconProxy.test.js

```diff
@@ -1,55 +1,105 @@
 const { ethers } = hre;
 const assert = require('assert');
+const assertRevert = require('@synthetixio/core-js/utils/assert-revert');
+const { findEvent } = require('@synthetixio/core-js/utils/events');
 
 describe('BeaconProxy', () => {
-  let Beacon, BeaconProxy, MockedBeaconProxy, Implementation;
+  let BeaconProxy, Instance, Beacon, Implementation;
```

Comment:
This is almost identical to UUPSProxy.test.js, except that the implementation is set in the beacon.